rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml

```diff
@@ -21,6 +21,22 @@ funcs:
 tenets:
   - name: comment-first-word-when-empty
     actions:
+ codelingo/docs:
+        title: Comment First Word as Subject
+        body: |
+          Each exported function should have a doc comment with a summary of the function
+          to allow for an automated presentation of these doc comments.
+          The first word in the summary should be the function name, followed by a description. 
+          
+          ## Bad code
+          ```
+          func Unnamed(){}
+          ```
+          ## Good code
+          ```
+          //Unnmaded is the first word and this is a summary of the function. 
+          func Unnamed(){}
```

Comment:
Small typo here and a single space is normal between the `//` and the comment.

I'd also change the examples to use something more realistic, and then layout the function like:
```
func Decrypt(data []byte) {
   ...
}
```